examples/src/main/scala/org/apache/spark/examples/mllib/MovieLensALS.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.log4j.{Level, Logger}
 import scopt.OptionParser
 
 import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.internal.config.Kryo._
```

Comment:
Hm, I think we might not want to modify the examples actually. They demonstrate what users might run. But they won't be able to import internal classes like this, no?

Reply:
@srowen That make sense, revert it.